raster-foundry/raster-foundry
# Add color scheme dropdown to lab histograms

## Overview
Fix max zoom level of lab layers
Add viridis to color maps and use as default for all nodes

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo
![image](https://user-images.githubusercontent.com/4392704/30385406-0762368c-9875-11e7-8d0d-5dfe064828ce.png)
![image](https://user-images.githubusercontent.com/4392704/30385415-0d959030-9875-11e7-9bec-72baeca7d168.png)
![image](https://user-images.githubusercontent.com/4392704/30385431-203cb128-9875-11e7-90a3-6658db14757b.png)


### Notes
We aren't sure how we should be implementing the continuous / bin options in the drop down, so I've punted on that for now
For now, I expect that we will be running anything that needs to be classified through a classification node.

## Testing Instructions
* Verify that the color scheme selector works, and that all color schemes are treated as continuous.
* Verify that the selection stays when you open/close the histogram

Closes #2492
